css/style.css

```
@@ -362,6 +362,8 @@ html {
   background-repeat: repeat;
   padding-top: 10rem;
   padding-bottom: 6.4rem;
+  padding-left: 28rem;
+  padding-right: 28rem;
```

Why not use the property shorthand?
```suggestion
padding: 10rem 28rem 6.4rem;
```

---

fixed